--- config.h.original	2013-02-25 13:11:03.628000004 +0000
+++ config.h	2013-02-25 13:11:23.260000004 +0000
@@ -199,10 +199,10 @@
 #define HAVE_LOCALE_H 1
 
 /* Define to 1 if the system has the type `long double'. */
-#define HAVE_LONG_DOUBLE 1
+#define HAVE_LONG_DOUBLE 0
 
 /* Define to 1 if the system has the type `long long'. */
-#define HAVE_LONG_LONG 1
+#define HAVE_LONG_LONG 0
 
 /* Define to 1 if you have the <machine/hal_sysinfo.h> header file. */
 /* #undef HAVE_MACHINE_HAL_SYSINFO_H */
@@ -343,7 +343,7 @@
 #define HAVE_NL_TYPES_H 1
 
 /* Define to 1 if you have the `obstack_vprintf' function. */
-#define HAVE_OBSTACK_VPRINTF 1
+#define HAVE_OBSTACK_VPRINTF 0
 
 /* Define to 1 if you have the `popen' function. */
 #define HAVE_POPEN 1
@@ -362,7 +362,7 @@
 #define HAVE_PTRDIFF_T 1
 
 /* Define to 1 if the system has the type `quad_t'. */
-#define HAVE_QUAD_T 1
+#define HAVE_QUAD_T 0
 
 /* Define to 1 if you have the `raise' function. */
 #define HAVE_RAISE 1
