--- CGAL_SetupBoost.cmake.original	2013-02-25 17:28:54.220000001 +0000
+++ CGAL_SetupBoost.cmake	2013-02-25 17:29:12.856000000 +0000
@@ -3,7 +3,7 @@
   include(CGAL_TweakFindBoost)
   # In the documentation, we say we require Boost-1.39, but technically we
   # require 1.33.1. Some packages may require more recent versions, though.
-  find_package( Boost 1.33.1 REQUIRED thread system )
+  
   
   message( STATUS "Boost include:     ${Boost_INCLUDE_DIRS}" )
   message( STATUS "Boost libraries:   ${Boost_LIBRARIES}" )
