--- Interval_nt.h.original	2013-03-01 16:13:37.336000003 +0000
+++ Interval_nt.h	2013-03-01 16:17:29.184000003 +0000
@@ -156,10 +156,12 @@
       // on the second argument, so if -fno-rounding-math, the compiler optimizes
       // the 2 negations and we get wrong rounding.
       typename Interval_nt<>::Internal_protector P;
+#if 0
       CGAL_assertion_msg(-CGAL_IA_MUL(-1.1, 10.1) != CGAL_IA_MUL(1.1, 10.1),
                          "Wrong rounding: did you forget the  -frounding-math  option if you use GCC (or  -fp-model strict  for Intel)?");
       CGAL_assertion_msg(-CGAL_IA_DIV(-1., 10) != CGAL_IA_DIV(1., 10),
                          "Wrong rounding: did you forget the  -frounding-math  option if you use GCC (or  -fp-model strict  for Intel)?");
+#endif    
     }
   };
 
